plugins/module_utils/v3/entity.py

```diff
@@ -179,6 +179,15 @@ def list(
             no_response=no_response,
             timeout=timeout,
         )
+        if resp and resp.get("state") == "ERROR":
```

Comment:
how will this impact backward compatibility with info modules ? Do we need to document in change in raising errors ?